tests/lib/rook/base.py

```diff
@@ -65,7 +65,7 @@ def execute_in_ceph_toolbox(self, command, log_stdout=False):
             self.toolbox_pod = toolbox_pods[0]
 
         return self.kubernetes.execute_in_pod(
-            command, self.toolbox_pod, log_stdout=False)
+            command, self.toolbox_pod, log_stdout=True)
```

Comment:
This looks like a mistake. It should be `log_stdout`, ie, the parameter on `execute_in_ceph_toolbox`.

It likely defaults to False because it is noisy, but I'm only guessing. It could also be from copying elsewhere.

Reply:
Yep this looks like a mistake both in the initial code and a forgotten debug for me.
Do you want to fix it here or shall we do another PR ?

Reply:
The change is unrelated, so put into a separate PR with explanation why we need this.

Reply:
Let's do a separate PR then